### PR TITLE
Llm integration

### PR DESCRIPTION
**Pull request by TanyaFolloze**

### CODE REVIEW

**Comment by TanyaFolloze** [comment]
Task linked: [FLZ-7957 Generate text for single section](https://app.clickup.com/t/3749831/FLZ-7957)

File: src/common/interfaces/IEvent.ts [hunk 1]
```diff
@@ -91,4 +91,5 @@ export type FLZ_DESIGNER_EVENT_ACTION =
     | "get-designer-themes"
     | "navigate-to-documentation"
     | "open-generic-dialog"
-    | "custom-sections-action";
+    | "custom-sections-action"
+    | "generate-section-text";
```
Comment (rotemgrim) [inline src/common/interfaces/IEvent.ts:95]: change to something like "generate-ai-action" to include all other future events (with a switch case)

File: src/common/interfaces/IInitialState.ts [hunk 2]
```diff
@@ -39,6 +39,7 @@ export type InitialState = {
         multiPages?: boolean;
         recommendationsAi: boolean;
         customTheme: boolean;
+        genAi: boolean;
```
Comment (rotemgrim) [inline src/common/interfaces/IInitialState.ts:42]: check that its always exists on the server

**Review by rotemgrim — CHANGES_REQUESTED** [review]

**Comment by TanyaFolloze** [comment]
@rotemgrim some github voodo happened, i re-created the pr https://github.com/folloze/client-sdk/pull/136
i renamed the branch but github didnt update for some weird reason and it didnt get my pr changes
im closing this one

**Pull request closed without merging**